### PR TITLE
Completion: Capitalize short desc and remove extra space from long 

### LINKED ISSUE #1458: Default `help` and `completion` command descriptions are inconsistent
As of v1.2.0, there are two commands that are included by default in every Cobra CLI. Their formatting is inconsistent, which is incredibly sloppy, especially considering that their descriptions can't be changed; to my knowledge these commands can only be hidden.

First, one `Short` description is uppercased, and one is lowercased. Recommendation: Make them both uppercase.
```
Available Commands:
  completion  generate the autocompletion script for the specified shell
  help        Help about any command
```

Second, one `Long` description has an extra leading newline, and the other does not. Recommendation: Remove the extra newline.
```
$ cmd completion -h

Generate the autocompletion script for spotify for the specified shell.
See each sub-command's help for details on how to use the generated script.

...
```

```
$ cmd help -h
Help provides help for any command in the application.
Simply type cmd help [path to command] for full details.

...
```

In conclusion, it looks like the new `completion` command did not even attempt to maintain consistent formatting. A good long term solution would be to allow users to edit these descriptions to match their preferred style.

### PR DESCRIPTION
**Pull request by fcjr**
The `help` short desc is capitalized so we should follow its lead for consistency.

Also fixes #1458.

### CODE REVIEW

**Review by marckhouzam — APPROVED** [review]
Thanks @fcjr.  You are right, the `completion` command should follow the pattern of the `help` command.

I assume there is no backwards-compatibility concerns about a command description or help text, but I'll leave that up to the maintainers.

**Review by umarcor — APPROVED** [review]

**Comment by brianstrauch** [comment]
As per #1458, the `Long` description is also inconsistent. Can we add a fix for that to this PR?

**Comment by fcjr** [comment]
> As per #1458, the `Long` description is also inconsistent. Can we add a fix for that to this PR?

Updated to fix the extra whitespace in the long desc.

**Comment by fcjr** [comment]
Just a friendly ping on this, is there any chance this can be merged and released?

**Comment by fcjr** [comment]
Bumping b/c it was marked as stale =)

**Review by stmcginnis — APPROVED** [review]
This looks good to me.

**Review by invidian — APPROVED** [review]
Cool! LGTM

**Review by jpmcb — APPROVED** [review]

**Pull request merged**
